sintia/sintia.py

```diff
@@ -568,6 +568,13 @@ def format_quote(self, quote: Quote) -> str:
             'hPa': {
                 'inHg': lambda inHg: inHg / 33.8639,
             },
+            'meters': {
+                'centimeter': lambda cm: cm * 100,
+                'inch': lambda in: in * 39.37,
```

Comment:
`in` is a reserved word in python, which means you can't name a variable with it. You could use `inch` instead.